modules/ROOT/pages/sharepoint/sharepoint-connector.adoc

```diff
@@ -56,6 +85,53 @@ Configuring the connector to use Okta authentication requires the following info
 * Okta API Token: Okta uses a bearer token for API authentication with a sliding scale expiration, for example: `+http://developer.okta.com/docs/api/getting_started/getting_a_token.html+`
 * SharePoint Online Embedded Link: The link of your Microsoft Office 365 SharePoint Online application from Okta, for example: `+https://your-domain.okta.com/home/office365/0oa14mz7f4HAWypdZ1t7/31737+`
 
+== Security Token Authentication
+
+You can use a SAML security token to log into SharePoint.
```

Comment:
GLOBAL:

to login to > to log in to
(login is an adjective or a noun, but not a verb)

Reply:
Substitute colon for period.